src/SignalR/clients/ts/client-ts.npmproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <IsPackable>false</IsPackable>
     <IsBuildable>false</IsBuildable>
-    <IsTestProject>false</IsTestProject>
+    <IsTestProject>true</IsTestProject>
```

Comment:
Huh? Doesn't look like a test project

Reply:
It is, it runs unit tests for /signalr and /signalr-protocol-msgpack